appIndicator.js

```diff
@@ -326,6 +326,11 @@ class AppIndicators_IconActor extends St.Icon {
 
         Util.connectSmart(Gtk.IconTheme.get_default(), 'changed', this, '_invalidateIcon')
 
+        Util.connectSmart(this, 'enter-event', this, () => {
+            this.opacity = 255;
```

Comment:
Didn't you prefer the methods? :smile:

Reply:
This is already the case, changed with later commit "appIndicator: Use setter for opacity".